man/systemd-nspawn.xml

```diff
@@ -980,6 +980,20 @@
         effect.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>--notify-mode=</varname></term>
+
+        <listitem><para>Configures support for notifications from init process in the container.
+        <varname>--notify-mode=</varname> supports three options: <option>no</option>, <option>yes</option>, and <option>ready</option>.
+        Option <option>no</option> do not enable notifications between systemd-nspawn and
+        init process in the container and systemd-nspawn notifies systemd with a <literal>READY=1</literal>
+        message when the init process is created. Option <option>yes</option> enables notifications
+        but do nothing with them, systemd-nspawn notifies systemd with a <literal>READY=1</literal>
+        message when the init process is created. Option <option>ready</option>
+        enables notifications and wait for the <literal>READY=1</literal> message from
+        the init process in the container before sending its own to systemd.</para></listitem>
+      </varlistentry>
```

Comment:
and again, i don't think that propagation of STATUS= should be conditional. Just forward that always, I see no reason why we would make that optional. I think propagating STATUS is entirely safe, and there's no reason to ever disable that.  Propagating READY= is something else though, and propagating completely arbitrary fields, too. Hence, have individual settings:

 ```
NotifyReady=true
PropagateNotify=X_FOO X_BAR X_WALDO
```